web/src/ferrostar-map.ts

```diff
@@ -347,11 +347,28 @@ export class FerrostarMap extends LitElement {
         "line-cap": "round",
       },
       paint: {
-        "line-color": "#3700B3",
+        "line-color": "#0099ff",
```

Comment:
Found the color from iOS using a screen color meter:

```suggestion
        "line-color": "#3478f6",
```